src/System.IO.FileSystem/src/System/IO/FileStatus.Unix.cs

```diff
@@ -113,6 +113,26 @@ public void SetAttributes(string path, FileAttributes attributes)
             if (!_exists)
                 FileSystemInfo.ThrowNotFound(path);
 
+            if (Interop.Sys.CanSetHiddenFlag)
+            {
+                if ((attributes & FileAttributes.Hidden) != 0)
+                {
+                    if ((_fileStatus.UserFlags & (uint)Interop.Sys.UserFlags.UF_HIDDEN) != (uint)Interop.Sys.UserFlags.UF_HIDDEN)
```

Comment:
Is this style terser
`if ((_fileStatus.UserFlags &(uint)Interop.Sys.UserFlags.UF_HIDDEN) == 0)`

Reply:
Yep, it might. Will change.

Reply:
In that case you might also change `                if ((_fileStatus.UserFlags & (uint)Interop.Sys.UserFlags.UF_HIDDEN) == (uint)Interop.Sys.UserFlags.UF_HIDDEN)` to `                if ((_fileStatus.UserFlags & (uint)Interop.Sys.UserFlags.UF_HIDDEN) != 0)`